src/v2i-hub/PedestrianPlugin/src/FLIRWebSockAsyncClnSession.cpp

```diff
@@ -227,80 +239,87 @@ namespace PedestrianPlugin
-                lat.erase(std::remove(lat.begin(), lat.end(), '.'), lat.end());
-                lat.pop_back();
+                xOffset = std::to_string(xOffsetInt);
             }
-
-            // Parse longitude
-            if (!it.second.get_child("longitude").data().empty())
+            if (!it.second.get_child("y").data().empty())
             {
+                int yOffsetInt = std::round(std::stod(it.second.get_child("y").data())*10);
+
                 // Convert received lat/lon to J2735 lat/lon format
-                lon = it.second.get_child("longitude").data();
-                lon.erase(std::remove(lon.begin(), lon.end(), '.'), lon.end());
-                lon.pop_back();
+                yOffset = std::to_string(yOffsetInt);
             }
 
+
             // Parse speed
             if (!it.second.get_child("speed").data().empty())
             {
                 // Speed from the FLIR camera is reported in m/s
                 // Convert to units of 0.02 m/s for J2735 Speed and Velocity format
                 float rawSpeed = std::stof(it.second.get_child("speed").data());
                 speed = static_cast<int>(std::round(rawSpeed / 0.02f));
-            }
-
-            // Parse out seconds from datetime string
-            std::vector<int> dateTimeArr = timeStringParser(time);                       
+            }                  
 
-            msgCount += 1;
-            if (msgCount > 127) {
-                msgCount = 0;
+            if (generateSDSM_ == false)
+            {
+                msgCount += 1;
+                if (msgCount > 127) {
+                    msgCount = 0;
+                }
             }
 
-            PLOG(logINFO) << "Received FLIR camera data at: " << dateTimeArr[0] << "/" << dateTimeArr[1] << "/" << dateTimeArr[2]
-                          << " " << dateTimeArr[3] << ":" << dateTimeArr[4] << ":" << dateTimeArr[5] << ":" << dateTimeArr[6];  
+            PLOG(logINFO) << "Message count: " << msgCount;
+            PLOG(logDEBUG) << "Sent XMLs to BroadcastPedDet: ";  
 
-            PLOG(logINFO) << "Received FLIR camera data for pedestrian " << idResult << " at location: (" << lat << ", " << lon <<
+            PLOG(logINFO) << "Received FLIR camera data for pedestrian " << idResult << " at location: (" << xOffset << ", " << yOffset <<
                           "), traveling at speed: " << speed << ", with heading: " << alpha << " degrees";
 
-            PLOG(logINFO) << "Message count: " << msgCount;
-            PLOG(logDEBUG) << "Sent XMLs to BroadcastPedDet: ";
-
             std::lock_guard<mutex> lock(_msgLock);
             if (generatePSM_ == true)
             {
+                // Convert Object ID to 4 octet string
+                std::stringstream idstream;
+                idstream << std::hex << id;
+                auto objectTempId = idstream.str();
+                int str_length_diff = 8 - static_cast<int>(objectTempId.length());
```

Comment:
Please use consistent variable case. `str_length_diff `to `strLengthDiff`?

Reply:
Will do